src/components/ui/form-textfield/FormTextfield.stories.ts

```diff
@@ -0,0 +1,89 @@
+import FormTextfield from './FormTextfield.vue'
+import type { Meta, StoryObj } from '@storybook/vue3'
+
+/** A text input field is a space where users can type and enter short pieces of information, such as names, addresses, or other brief text. */
+const meta: Meta<typeof FormTextfield> = {
+	title: 'Ui/FormTextfield',
```

Comment:
no título podemos colocar dentro de um grupo dentro do storybook
```suggestion
	title: 'Ui/Form/Textfield',
```

acredito que também podemos ir adicionando isso aos futuros componentes adicionados e, tambem aos que ainda nao tem

Reply:
Excelente ponto Vitor!

Reply:
Foi fazer isso no pr https://github.com/uxshop/design-system/pull/194/files que está a frente desse, ok?